src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVEDescription.hpp

```diff
@@ -0,0 +1,130 @@
+        std::string VulnDescFBDescriptionStr;
+        std::string VulnDescFBSeverityStr;
+        std::string VulnDescFBScoreVersionStr;
+        std::string VulnDescFBReferenceStr;
+
+        bool cont = true;
+        for (size_t i = 0; i < sizeT; ++i)
+        {
+            if (i < sizeM)
+            {
+                auto metric = metricsArray->Get(i);
+                if (metric->cvssV3_1())
+                {
+                    VulnDescFBScoreBase = metric->cvssV3_1()->baseScore();
+                    auto baseSeverity = metric->cvssV3_1()->baseSeverity();
+                    VulnDescFBSeverityStr = (nullptr != baseSeverity) ? baseSeverity->str() : "";
+                    auto version = metric->cvssV3_1()->version();
+                    VulnDescFBScoreVersionStr = (nullptr != version) ? version->str() : "";
+                    auto format = metric->format();
+                    VulnDescFBClassificationStr = (nullptr != format) ? format->str() : "";
+                }
+            }
+            if (i < sizeR)
+            {
+                auto reference = referencesArray->Get(i);
+                if (reference->url())
+                {
+                    VulnDescFBReferenceStr = VulnDescFBReferenceStr + reference->url()->str() + ", ";
+                }
+            }
+            if ((i < sizeD) && cont)
+            {
+                auto description = descriptionArray->Get(i);
+                if (description->lang()->str().compare("en") == 0)
+                {
+                    VulnDescFBDescriptionStr = description->value()->str();
+                    cont = false;
+                }
+            }
+        }
+        VulnDescFBReferenceStr = VulnDescFBReferenceStr.substr(0, VulnDescFBReferenceStr.size() - 2);
+
+        flatbuffers::FlatBufferBuilder builder;
+
+        auto VulnDescFBClassification {builder.CreateString(VulnDescFBClassificationStr)};
+        auto VulnDescFBDescription {builder.CreateString(VulnDescFBDescriptionStr)};
+        auto VulnDescFBSeverity {builder.CreateString(VulnDescFBSeverityStr)};
+        auto VulnDescFBScoreVersion {builder.CreateString(VulnDescFBScoreVersionStr)};
+        auto VulnDescFBReference {builder.CreateString(VulnDescFBReferenceStr)};
+
+        NSVulnerabilityScanner::VulnerabilityDescriptionBuilder vulnerabilityDescriptionBuilder(builder);
+        vulnerabilityDescriptionBuilder.add_scoreBase(VulnDescFBScoreBase);
+        vulnerabilityDescriptionBuilder.add_classification(VulnDescFBClassification);
+        vulnerabilityDescriptionBuilder.add_description(VulnDescFBDescription);
+        vulnerabilityDescriptionBuilder.add_severity(VulnDescFBSeverity);
+        vulnerabilityDescriptionBuilder.add_scoreVersion(VulnDescFBScoreVersion);
+        vulnerabilityDescriptionBuilder.add_reference(VulnDescFBReference);
+
+        auto vulnerabilityDescriptionFB = vulnerabilityDescriptionBuilder.Finish();
+        builder.Finish(vulnerabilityDescriptionFB);
```

Comment:
Is it ok to `Finish()` twice?

Reply:
I used this guide https://flatbuffers.dev/flatbuffers_guide_tutorial.html

```
auto orc = monster_builder.Finish();
...
// Call `Finish()` to instruct the builder that this monster is complete.
// Note: Regardless of how you created the `orc`, you still need to call
// `Finish()` on the `FlatBufferBuilder`.
builder.Finish(orc); // You could also call `FinishMonsterBuffer(builder, orc);`.
```

Reply:
Ok, thank you